### ISSUE TITLE
Spawn diamonds in predefined patterns, at least sometimes

### ISSUE THREAD

**Issue by indrif** [Etimo/diamonds#62]
Or perhaps predefine a list of "random" patterns that we switch between when spawning diamonds. Some of these could be cool patterns such as E T I M O, just for fun if anyone sees :)

**Comment by teozkr** [comment]
By the way, wouldn't this conflict with variable-size boards? I guess we we could just spawn it in the middle of a bigger board...

**Comment by indrif** [comment]
Yes it would if we were to spawn diamonds in the form of some predefined text. However we could also just spawn diamonds in predefined patterns that take the board dimension as input, such as "square in the middle", "straight line across the board" etc etc.

But this is not very well thought through and probably not something that will cause exciting implementations.
